Assets/Plugins/Editor/Uplift/LogHandler.cs

```diff
@@ -0,0 +1,53 @@
+using UnityEngine;
+using System.Collections;
+using System.IO;
+using System.Text;
+using System;
+
+public class LogHandler : IDisposable
+{
+	public bool ShowStackTrace = false;
+	public string LogFileName = "log.txt";
+
+	private string output = "";
+	private string stack = "";
+
+	private StreamWriter OutputStream;
+
+	public LogHandler(string logFileName = "uplift.log", bool appendToCurrentLogFile = false, bool showStackTrace = false)
+	{
+		ShowStackTrace = showStackTrace;
+		LogFileName = logFileName;
+
+		OutputStream = new StreamWriter(logFileName, appendToCurrentLogFile);
+
+		Application.logMessageReceived += HandleLog;
+	}
+
+	public void Dispose()
+	{
+		Application.logMessageReceived -= HandleLog;
+
+		if (OutputStream != null)
+		{
+			OutputStream.Close();
+			OutputStream = null;
+		}
+	}
+
+	void HandleLog(string logString, string stackTrace, LogType type)
```

Comment:
We lose the `type` information. Not sure it's super important but maybe we want to prefix the logfile with something like `[type]`, `type.firstCharacter` or something like that